packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -596,6 +602,7 @@ export class GarbageCollector implements IGarbageCollector {
 
         // Whether we are running in test mode. In this mode, unreferenced nodes are immediately deleted.
         this.testMode = this.mc.config.getBoolean(gcTestModeKey) ?? this.gcOptions.runGCInTestMode === true;
+        this.tombstoneMode = this.mc.config.getBoolean(tombstoneKey) ?? this.gcOptions.tombstoneMode === true;
```

Comment:
Please add unit tests similar to the tests for "testMode"

Reply:
Will add when we come up with a clear decision on what we want to config tombstone mode as